api-catalog-ui/frontend/src/utils/utilFunctions.js

```diff
@@ -30,13 +30,32 @@ export default function countAdditionalContents(service) {
     return { useCasesCounter, tutorialsCounter, videosCounter };
 }
 
+function setButtonsColor(wizardButton, uiConfig, refreshButton) {
+    if (wizardButton) {
+        if (uiConfig.headerColor === 'white' || uiConfig.headerColor === '#FFFFFF') {
+            wizardButton.style.setProperty('color', 'black');
+        } else {
+            wizardButton.style.setProperty('color', uiConfig.headerColor);
+        }
+    }
+    if (refreshButton) {
+        if (uiConfig.headerColor === 'white' || uiConfig.headerColor === '#FFFFFF') {
+            refreshButton.style.setProperty('color', 'black');
+        } else {
+            refreshButton.style.setProperty('color', uiConfig.headerColor);
+        }
+    }
```

Comment:
```suggestion
    const color = (uiConfig.headerColor === 'white' || uiConfig.headerColor === '#FFFFFF') ? 'black' : uiConfig.headerColor;
    if (wizardButton) {
        wizardButton.style.setProperty('color', color);
    }
    if (refreshButton) {
        refreshButton.style.setProperty('color', color);
    }
```

Make the condition really sense? Replace white with black otherwise use the same...